src/content/go/database_sql/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "database/sql"
+date: 2022-12-02T12:56:00+05:30
+draft: false
+weight: 1
+tags: ["go", "sql"]
+---
+
+SQLCommenter provides drop-in replacement for [go's `database/sql`](https://pkg.go.dev/database/sql) library. You can start using the [SQLcommenter's `database/sql`](https://github.com/google/sqlcommenter/tree/master/go/database/sql) without changing your business logic code and use one of the frmaework instrumentation to proapagate tags and traceparent to the SQL queries.
+
+## Installation
+
+```shell
+go get -u github.com/google/sqlcommenter/go/database/sql
+```
+
+## Usage
+
+Please use the sqlcommenter's default go-sql database driver to execute statements. 
+Due to inherent nature of Go, the safer way to pass information from framework to database driver is via `context`. So, it is recommended to use the context based methods of `DB` interface like `QueryContext`, `ExecContext` and `PrepareContext`. 
+
+```go
+import (
+    gosql "github.com/google/sqlcommenter/go/database/sql"
+    sqlcommentercore "github.com/google/sqlcommenter/go/core"
+    _ "github.com/lib/pq" // or any other database driver
+)
+
+db, err := gosql.Open("<driver>", "<connectionString>", sqlcommentercore.CommenterOptions{
```

Comment:
instead of using :=, we can do this instead to make it more clear ?

```
var (
  db *sql.DB
  err error
)

db, err = gosql.Open("<driver>", "<connectionString>", sqlcommentercore.CommenterOptions{...})
```

Reply:
Also, you need to then import `database/sql`.